packages/core/src/connectivity/scp-cf/environment-accessor.ts

```diff
@@ -203,7 +204,7 @@ export function getDestinationServiceUri(): string | null {
  * @returns The credentials for a match, otherwise null.
  */
 export function getXsuaaServiceCredentials(
-  token?: DecodedJWT | string
+  token?: JwtPayload | string
```

Comment:
Question: What would happen here if a customer used the `getXsuaaServiceCredentials` and typed the response with `DecodedJwt` is the strctural typing so flexible that the types `JwtPayload` and `DecodedJwt` are competabile? If not we should add a note.

Reply:
I think `JwtPayload` and `DecodedJwt` are compatible, as the `JwtPayload` can be any object:
```
export interface JwtPayload {
  [key: string]: any;
  iss?: string;
  sub?: string;
  aud?: string | string[];
  exp?: number;
  nbf?: number;
  iat?: number;
  jti?: string;
}
```

Reply:
Do you think it make sense to also create some type tests? For example if next time the interface says the `iss` becomes mandatory instead of optional like below, I will detect it from the type test and I know `JwtPayload` and `DecodedJwt` are not compatible. Of course, as we have compatibility notes, I'm fine to stop considering `DecodedJwt` after this release.
```
export interface JwtPayload {
  iss: string;
...
}
```

Reply:
The three replaced interfaces are compatible, when using the SDK. The only difference is, if someone was using the old `JWTHeader`, where `typ` was a mandatory property, which is now optional. If they were using this in their own code somehow and rely on the `typ` to be there, they would have to add a check for undefined before.